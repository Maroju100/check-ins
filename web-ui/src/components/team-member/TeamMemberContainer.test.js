import React from "react";
import TeamMembercontainer from "./TeamMemberContainer";

const testProfile = [
  { name: "holmes", image_url: "" },
  { name: "homie", image_url: "" },
];

it("renders correctly", () => {
  snapshot(<TeamMembercontainer profiles={testProfile} />);
});
