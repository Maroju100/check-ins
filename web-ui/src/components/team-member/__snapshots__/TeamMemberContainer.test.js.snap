// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <div
    className="flex-row"
    style={
      Object {
        "flexWrap": "wrap",
      }
    }
  >
    <div
      className="image-div"
      onClick={[Function]}
    >
      <img
        alt="Profile"
        className="member-image"
        src="https://i.imgur.com/TkSNOpF.jpg"
      />
    </div>
    <div
      className="image-div"
      onClick={[Function]}
    >
      <img
        alt="Profile"
        className="member-image"
        src="https://i.imgur.com/TkSNOpF.jpg"
      />
    </div>
  </div>
</div>
`;
